src/Bab/RabbitMq/VhostManager.php

```diff
@@ -180,14 +180,19 @@ private function createQueues(Configuration $config)
             }
 
             for ($i = 0; $i < count($retries); $i++) {
+                if ($i === 0) {
```

Comment:
this still creates the exchange once for each queue needing retry. The exchange should be created only the first time (and then binding should be created for the first iteration of this loop